reproducibility_project/src/engines/hoomd/project.py

```diff
@@ -36,29 +34,69 @@ def add_args(cls, parser):
         parser.add_argument("--nodelist", help="Specify the node to submit to.")
 
 
-# This environment variable is set by running
+# The MOSDEF_PYTHON environment variable is set by running
 # echo "export MOSDEF_PYTHON=$(which python)" >> ~/.bashrc
 # with the mosdef-study38 conda env active
 @Project.operation.with_directives({"executable": "$MOSDEF_PYTHON", "ngpu": 1})
```

Comment:
just so i understand, what is the importance of this section with the env variable?

Reply:
in order to use a conda environment on our cluster, we have to provide the path to that python executable--for me this would be something like `/home/jennyfothergill/... ` our conda environment python executables are in our user directory. If I set this code to run on my specific conda installation and @chrisjonesBSU wanted to also run this project, he would have to specify a different path. By using this environment variable which is set in a standard way, we can each set it separately.